packages/astro/test/hosted/README.md

```diff
@@ -0,0 +1,3 @@
+The tests in this folder are done on remote hosts (Netlify and Vercel), and are not run by the test suite. They get ran every week through a GitHub action.
+
+The purpose of those test is to make sure that everything works as expected while deployed, in a way, they're as E2E as it gets.
```

Comment:
I believe this is the only information for docs to review? 

Just some small punctuation, verb agreement, and typo nits 🙌 
```suggestion
The tests in this folder are done on remote hosts (Netlify and Vercel) and are not run by the test suite. They ran every week through a GitHub action.

The purpose of those tests is to make sure that everything works as expected while deployed. In a way, they're as E2E as it gets.
```